assets/js/components/notifications/ZeroDataStateNotifications.js

```diff
@@ -113,28 +113,26 @@ export default function ZeroDataStateNotifications() {
 				/>
 			) }
 
-			{ analyticsGatheringData === false &&
-				searchConsoleGatheringData === false &&
-				( analyticsHasZeroData || searchConsoleHasZeroData ) && (
-					<BannerNotification
-						id="zero-data-notification"
-						title={ __(
-							'Not enough traffic yet to display stats',
-							'google-site-kit'
-						) }
-						description={ __(
-							'Site Kit will start showing stats on the dashboard as soon as enough people have visited your site. Keep working on your site to attract more visitors.',
-							'google-site-kit'
-						) }
-						format="small"
-						learnMoreLabel={ __( 'Learn more', 'google-site-kit' ) }
-						learnMoreURL="https://sitekit.withgoogle.com/documentation/using-site-kit/dashboard-data-display/"
-						dismiss={ __( 'Remind me later', 'google-site-kit' ) }
-						isDismissible
-						dismissExpires={ getTimeInSeconds( 'day' ) }
-						SmallImageSVG={ ZeroStateIcon }
-					/>
-				) }
+			{ ( analyticsHasZeroData || searchConsoleHasZeroData ) && (
```

Comment:
Hi @hussain-t, the problem with removing this condition is that it's now possible for _both_ notifications to be displayed when a module is in both "gathering data" and "zero data" state, which is possible - in this case, only the "gathering data" notification should be displayed.

I think the condition here should be quite specific, checking each module to ensure it's _not_ in "gathering data" state, but _is_ in the "zero data" state. In other words:

```js
			{ ( analyticsGatheringData === false && analyticsHasZeroData )
						|| ( searchConsoleGatheringData &&	searchConsoleHasZeroData ) && (
```

Reply:
Thanks, @techanvil. The condition should be:
```
{ ( analyticsGatheringData === false && analyticsHasZeroData ) ||
	( searchConsoleGatheringData === false && searchConsoleHasZeroData ) && ( <BannerNotification {...props} /> ) }